src/main/rx/rx.h

```diff
@@ -114,6 +114,7 @@ typedef struct rxConfig_s {
     uint8_t spektrum_sat_bind;              // number of bind pulses for Spektrum satellite receivers
     uint8_t rssi_channel;
     uint8_t rssi_scale;
+    uint8_t rssi_warningLevel;
```

Comment:
is this correct? This will break configs in eeprom, at least version must be bumped. Also separate PR is reasonable ..